lib/shared-worker.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+/* global self */
+/* eslint no-eval: off */
+
+self.JSONBird = require('jsonbird');
```

Comment:
This will make `JSONBird` available to all mocha unit tests as a global right? This might cause someone's unit test to incorrectly pass or fail. For example: they might be using jsonbird in their own project but they forgot to import the library into the worker, so their tests should fail but instead they are incorrectly passing because they picked up our copy.

This file is compiled with browserify which wraps the whole file in an anonymous function, so we can just use `const` and `let` as if this file were a node module (without polluting the global scope). This is why I simply use `const JSONBird = require('jsonbird');` in `worker.js`.

Note that browserify passes `self` as `global` (which is what I used in worker.js). Either style (`self`/`global`) is fine with me though